src/auth/permissions/permissions.d.ts

```diff
@@ -0,0 +1,100 @@
+import * as Knex from 'knex';
+
+type Dictionary<T> = { [k: string]: T }
+
+type Role = 
```

Comment:
If these were enums we could export them and use them in calling code instead of hardcoding the values.